maintenance-mode.php

```diff
@@ -114,12 +114,12 @@ function vip_maintenance_mode_restrict_rest_api( $result ) {
 		return $result;
 	}
 
-	$error_message = apply_filters( 'vip_maintenance_mode_rest_api_error_message', __( 'REST API access is currently restricted while we refresh things a bit.', 'maintenance-mode' ) );
+	$error_message = apply_filters( 'vip_maintenance_mode_rest_api_error_message', __( 'REST API access is currently restricted, while this site is undergoing maintenance.', 'maintenance-mode' ) );
 	$unauthorized_error = new WP_Error(
-		'vip_maintenance_mode_rest_not_logged_in',
+		'vip_maintenance_mode_unauthorized',
```

Comment:
The new error code somewhat contradicts the new status code. The `401` status code is `Unauthorized`, which we are moving away from.

Reply:
Actually, looking at the conditions below, should we have separate errors for each?

Reply:
@mjangda Fair enough.  I'm presuming the first one would be a `401` for `is_user_logged_in()` and a `503` for `current_user_can_bypass_vip_maintenance_mode()`?

Reply:
@mjangda Any thoughts on ^^